### ISSUE TITLE
why randomly select one subject in data_loader.py

### ISSUE THREAD

**Issue by HJHjiahao** [longlongman/CasRel-pytorch-reimplement#12]
In line 72 of your data_loader.py, why only select one subject instead of performing subsequent operations on all subjects?
Thanks in advance!

**Comment by longlongman** [comment]
In the original code of CasRel model, the subject is also selected randomly.

**Comment by HJHjiahao** [comment]
Yeah, and why? I didn't find the theoretical basis of it in original paper. I am confused.  -_- Please!
